src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -513,7 +513,7 @@ public T[] ToArray()
         {
             // Short path if the bag is empty
             if (_headList == null)
-                return new T[0];
+				return Array.Empty<T>();
```

Comment:
@hackcraft, yeah, I think either we should take this change, or add such a test. Same for other collections.